downstream/modules/platform/con-aap-backup-recommendations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="aap-backup-recommendations"]
+
+= Backup recommendations
+
+
+[role="_abstract"]
+Recovering from data loss requires that you plan for and create backup resources of your {PlatformName} deployments on a regular basis. At a minimum, Red Hat recommends backing up deployments of {PlatformName} under the following circumstances:
+
+* Before upgrading your {PlatormName} deployments
```

Comment:
It looks like `{PlatormName}` didn't resolve correctly in the preview rendering fyi.  I think it instead needs to be `{PlatformName}`

Reply:
Fixed.